website/docs/r/container_cluster.html.markdown

```diff
@@ -195,13 +195,13 @@ The `ibm_container_alb` provides the following [Timeouts](https://www.terraform.
 Review the argument references that you can specify for your resource. 
 
 - `datacenter` - (Required, Forces new resource, String) The datacenter where you want to provision the worker nodes. The zone that you choose must be supported in the region where you want to create the cluster. To find supported zones, run `ibmcloud ks zones` [command line](https://cloud.ibm.com/docs/cli?topic=cloud-cli-getting-started).
-- `default_pool_size`  - (Optional, Integer) The number of worker nodes that you want to add to the default worker pool.
-- `disk_encryption` - (Optional, Forces new resource, Bool) If set to **true**, the worker node disks are set up with an AES 256-bit encryption. If set to **false**, the disk encryption for the worker node is disabled. For more information, see [Encrypted disks for worker node](https://cloud.ibm.com/docs/containers?topic=containers-security#workernodes).
+- `default_pool_size`  - (Optional, Integer) The number of worker nodes that you want to add to the default worker pool. Only affecting the cluster creation and has no other effect later, to modify the workerpool use its dedicated resource
+- `disk_encryption` - (Optional, Bool) If set to **true**, the worker node disks are set up with an AES 256-bit encryption. If set to **false**, the disk encryption for the worker node is disabled. For more information, see [Encrypted disks for worker node](https://cloud.ibm.com/docs/containers?topic=containers-security#workernodes). Only affecting the cluster creation and has no other effect later, to modify the workerpool use its dedicated resource.
```

Comment:
```suggestion
- `disk_encryption` - (Optional, Bool) If set to **true**, the disks for the workers in the default worker pool are set up with an AES 256-bit encryption, otherwise they are not encrypted. For more information, see [Encrypted disks for worker node](https://cloud.ibm.com/docs/containers?topic=containers-security#workernodes). This field only affects cluster creation, to manage the default worker pool, create a dedicated worker pool resource.
```